### PR TITLE
Upgrade some libs. nose->pytest

### PR DESCRIPTION
**Pull request by kosmos342**
Regular upgrading process

### CODE REVIEW

File: yolapi/yolapi/settings.py [hunk 1]
```diff
@@ -49,14 +51,12 @@
 
 SITE_ID = 1
 
+FORMAT_MODULE_PATH = ['yolapi.formats',]
```
Comment (snitch) [inline yolapi/yolapi/settings.py:54]: E231 missing whitespace after ','

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Review by RayeN — APPROVED** [review]

**Pull request merged**